tools/docker/sanity_check.Dockerfile

```diff
@@ -118,6 +118,7 @@ RUN --mount=type=cache,id=cache_bazel,target=/root/.cache/bazel \
     bash tools/install_so_files.sh
 RUN pip install --no-deps -e .
 RUN pytest -v -n auto ./tensorflow_addons/activations
+RUN pytest -v --doctest-modules tensorflow_addons/ --ignore-glob=*_test.py
```

Comment:
Is it possible to group everything in one call?

Reply:
Thanks @gabrieldemarmiesse.